viz-lib/src/visualizations/table/utils.js

```diff
@@ -62,6 +61,8 @@ export function prepareColumns(columns, searchInput, orderBy, onOrderByChange) {
       key: column.name,
       dataIndex: `record[${JSON.stringify(column.name)}]`,
       align: column.alignContent,
+      sorter: { multiple: 1 }, // using { multiple: 1 } to allow built-in multi-column sort arrows
```

Comment:
Just curious: is there any special meaning for constant `1`? It feels like it should be `true` :sweat_smile:

Reply:
Found an answer: https://github.com/ant-design/ant-design/blob/523b74e3b6a8be30b27e08643be246728e1b71dc/components/table/interface.tsx#L87-L88

If I understand this comment correctly, `multiple` field should be set to a sorting priority of the column. However, it looks like it works as it is now (with const `1`). Anyway, I leave it up to you. If you think it's important - you can try to fix it; or you can leave everything since it works.

Reply:
`true` didn't work 😕. I left it with const `1` because we are not using Antd's `compare` function, it's just the Icons.